member/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+from data_center.models import Course
+
+
+class Member(models.Model):
+    user = models.OneToOneField(User)
+
+    # Facebook field
+    uuid = models.CharField('User ID', max_length=100)
+    email = models.EmailField('Email')
+    access_token = models.TextField('access token')
```

Comment:
tokens will expire according to facebook
https://developers.facebook.com/docs/facebook-login/access-tokens